lib/thumbnail-sizes.php

```diff
@@ -9,11 +9,13 @@
   add_image_size( 'opengraph', 1200, 630, true );
   add_image_size( 'sponsor', 300, 100, false );
   add_image_size( 'splash', 1440, 550, true );
+
   add_image_size( 'col-3-crop', 252, 175, true );
   add_image_size( 'col-3-square', 252, 252, true );
   add_image_size( 'col-4-crop', 384, 230, true );
   add_image_size( 'col-6', 540, 9999, false );
   add_image_size( 'col-8', 770, 9999, false );
+  add_image_size( 'col-8-col-6', 770, 540, false );
```

Comment:
hmm. that also expresses as the ratio 4:3 which is kinda more common. I've been writing colx-4to3 as ratio sizes. A bit weird still but makes me think in common ratios when building which I think is good. A better syntax would be better tho